app/src/main/java/ch/blinkenlights/android/vanilla/PlaybackService.java

```diff
@@ -1174,18 +1174,19 @@ private void scrobbleBroadcast() {
 			intent.putExtra("track", song.title);
 			intent.putExtra("album", song.album);
 			intent.putExtra("artist", song.artist);
-			if (androidIds[0] != -1) {
-				intent.putExtra("songid", androidIds[0]);
-				intent.putExtra("albumid", androidIds[1]);
-			}
+			if (song.mbid != null) intent.putExtra("mbid", song.mbid);
 			sendBroadcast(intent);
 		}
 
 		if (mScrobble) {
 			Intent intent = new Intent("net.jjc1138.android.scrobbler.action.MUSIC_STATUS");
 			intent.putExtra("playing", (mState & FLAG_PLAYING) != 0);
-			if (androidIds[0] != -1)
-				intent.putExtra("id", androidIds[0]);
+			intent.putExtra("secs", (int)song.duration / 1000);
+			intent.putExtra("clashWithScrobbleDroid", "1");
```

Comment:
What does this line mean?

Reply:
https://github.com/simple-last-fm-scrobbler/sls/blob/294ecfef30b386e5e3d6cffc39a987f2bcdfe7fb/app/src/main/java/com/adam/aslfms/receiver/MusicAPI.java#L119

Reply:
Can you elaborate? I don't understand the functionality.

Reply:
It's just a boolean that the app can also use the unmaintained Scrobble Droid app https://github.com/JJC1138/scrobbledroid
It can be removed if requested, I guess most are using simple last fm scrobbler now.